client/views/view_nations_data.cpp

```diff
@@ -11,6 +11,11 @@
     \_____/ /                     If not, see https://www.gnu.org/licenses/.
       \____/        ********************************************************/
 
+/*
+ * This file contains functions used to gather varying data elements for use
+ * in the nations view (formally known as the plrdlg - player dialog).
```

Comment:
You can use Doxygen markers around file-level comments like this one, so the description appears in the generated docs (not that I've seen anyone use it though). [Relevant documentation.](https://www.doxygen.nl/manual/commands.html#cmdfile)